gunicorn.conf.py

```diff
@@ -1,6 +1,6 @@
 access_logfile = "-"
 error_logfile = "-"
-# max_requests = 500000  # Restart a worker after it has processed a given number of requests (for memory leaks)
+max_requests = 500000  # Restart a worker after it has processed a given number of requests (for memory leaks)
```

Comment:
We are changing this back because of the optimizations of this PR?

Reply:
Yes, this was impacting a lot memory usage on production